builders/interact/eth-libraries/deploy-contract.md

```diff
@@ -192,7 +184,7 @@ npm init --yes
 
 ### Web3.py {: #web3py } 
 
-In the first part of [the script](/snippets/code/web3py-contract/deploy.py), the `web3` instance (or provider) is created using the 在[此脚本](/snippets/code/web3py-contract/deploy.py)的第一部分，`web3`实例（或是提供者）是使用提供者RPC的`Web3(Web3.HTTPProvider(provider_rpc))`方式所创建的。您可以通过改变提供者RPC，选择要将交易发送到您指定的网络。
+In the first part of the script, the `web3` instance (or provider) is created using the 在此脚本的第一部分，`web3`实例（或是提供者）是使用提供者RPC的`Web3(Web3.HTTPProvider(provider_rpc))`方式所创建的。您可以通过改变提供者RPC，选择要将交易发送到您指定的网络。
```

Comment:
some of this is in English.. and it also looks like there might be an extra `web3` in there